_posts/2021-02-17-Out-With-The-Old.md

```diff
@@ -0,0 +1,178 @@
+* Experimenters: These organizations try to organize their innovation around their own resources, but try to limit the effort they expend for these initiatives. They tend to focus on low-cost techniques such as innovation training and design sprints in order to mature their internal resources and build a solid innovation capability and mindset. Spotify and Netflix are the success stories in this type of innovation.
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2021-02-17-Out-With-The-Old/innovationmatrix.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+##	Considerations for the Hunter Type
+
+Deciding which merger or acquisition to go for might seem as arbitrary as betting on which innovation horse will win the race, but there are consequences to this approach. 
+Clayton Christensen, father of the theory of disruptive innovation, pointed out that there are several valid reasons for joining two organizations (either by takeover or merger): either to improve current operations by boosting performance or reducing operational cost, and/or dramatically transform the organization’s growth prospects by reinventing the business model, and/or optimizing/diversifying the business portfolio.
+
+In any case, the business models of the two organizations will need to be integrated on a number of facets:
+* Customer Value Proposition: The services/products that customers value above the alternatives within the organization’s market segments.
+* Profit Formula: The revenue model and cost structure in place to ensure the organization is able to deliver the customer value proposition and generate the revenues needed to sustain the organization.
+* Organizational Resources: Any resources (employees, customers, products/services, infrastructure, cash flow…) that the organizations have to deliver the customer value proposition.
+* Organizational Processes: Some domains (manufacturing, R&D, budgeting, sales…) will be easier to match than others depending how much the views of the two organizations in these domains differ.
+
+This integration can happen in two distinct ways. 
+The first way is called the Leverage-my-Business-Model acquisition (LBM). 
+This is rather straightforward with all resources extracted from one of the organizations, and infused on the other one. 
+This incurs the least risk and thus the least potential for returns. 
+The current infrastructure of the master organization needs to be extended with the new components of the disappearing organization and it needs to scale its existing components to support the additional load. 
+The second way is called the Reinvent-my-Business-Model (RBM). 
+The business model of the acquired organization is used to revamp the existing business model of the master organization and create of fusion business model and infrastructure. 
```

Comment:
"... and create of fusion..." sounds a bit weird, is it meant to be "created of fusing"?